app/RequirementTests/RequirementTests.hs

```diff
@@ -0,0 +1,19 @@
+{-|
+Description: Requirement test suites module.
+
+Module that contains the test suites for all the requirement tests.
+
+-}
+
+module RequirementTests.RequirementTests
```

Comment:
I think you may have misunderstood my intentions with the file reorganization: I'd like `RequirementTests` to be a *subfolder* under `Tests` (so that all Haskell tests are under the same `Tests` folder).